.travis.yml

```diff
@@ -98,6 +103,41 @@ x-osx-39-shard: &x-osx-39-shard
     - *x-pyenv-env
     - *x-pyenv-39-env
 
+x-windows-shard: &x-windows-shard
+  os: windows
+  language: shell
+  cache:
+    # The default is 3 minutes (180).
+    timeout: 300
+    directories:
+      - .pyenv_test
+      - "${PYENV_ROOT}"
+  install:
+    - *x-tox-install
+    - *x-pyenv-install
+
+x-windows-27-shard: &x-windows-27-shard
+  <<: *x-windows-shard
+  before_install:
+  - choco install python2
```

Comment:
What's the thought behind using both choco to manage a python2 install here and a Python 3.9.0 install below and also setting up pyenv? One or the other would be fine, but both is not necessary for any obvious reason.

Reply:
I had copied it from the OSX shard but yes, it is indeed redundant. Removing `pyenv` from the Windows builds since it's a bit of a pain to deal with anyways.